etc/config/haskell.amazon.properties

```diff
@@ -5,13 +5,12 @@ supportsBinary=true
 supportsExecute=true
 objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 
+# GHC 8.2.2 & 8.6.3 had some problems that did not let us add them to the site
```

Comment:
As it says here, both of them gave us some errors when trying to use them so they got disabled and never reenabled 

(8.6.3 in https://github.com/compiler-explorer/compiler-explorer/commit/597e10042b49b614cb7cb45b0a5bee2e3abed5ae and 8.2.2 in https://github.com/compiler-explorer/compiler-explorer/commit/bffeb90d2dbc6c2ab1f488b11eac79d9e97798ca)

Reply:
Maybe should try to enable them and see if they now work, just in case?